ecs/scripts/generate-pr-to-plugins.sh

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env bash
+
+# Constants
+MAPPINGS_SUBPATH="mappings/v8.11.0/generated/elasticsearch/legacy/template.json"
```

Comment:
Hardcoded version

Reply:
Fixed in commit 67a71bdf9893ffc04da66130e9134440ef3a7229